core/modules/file/file.pages.inc

```diff
@@ -1,9 +1,388 @@
 <?php
 /**
  * @file
- * Supports file operations including View, Edit, and Delete.
+ * Supports file operations including Manage and Delete.
  */
 
+/**
+ * Page callback: Form constructor for the file edit form.
+ *
+ * @param object $file
+ *   A file object from file_load().
+ */
+function file_edit_form($form, &$form_state, $file) {
+  // @todo Add type w/ https://github.com/backdrop/backdrop-issues/issues/2632
+  backdrop_set_title(t('<em>Manage file</em> @title', array('@title' => $file->filename)), PASS_THROUGH);
```

Comment:
This title seems backwards. Don't we normally show the title in italics and the verb (Manage, Edit, Delete, etc) in the non-italicized font? `t('Manage file %name', array('%name' => $file->filename))`